core/src/main/web/app/mainapp/components/notebook/codecell-directive.js

```diff
@@ -379,6 +379,7 @@
           scope.cm = CodeMirror.fromTextArea(element.find('textarea')[0], codeMirrorOptions);
           scope.bkNotebook.registerCM(scope.cellmodel.id, scope.cm);
           scope.cm.on('change', changeHandler);
+          scope.cm.on('change', bkCoreManager.replaceTabsWithArrow);
```

Comment:
Maybe we can debounce this. If you're running this on every keystroke with a fast typist we could run into problems.

We use lodash, lodash has debounce: https://lodash.com/docs#debounce